src/components/modal/modal.scss

```diff
@@ -0,0 +1,51 @@
+@import '~theme/base/variables';
+@import '~theme/base/mediaqueries';
+
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width:100%;
+    height: 100%;
+    background: rgba($color-black, .3);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.innerModal {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    padding: 40px 20px;
+    overflow-y: auto;
```

Comment:
please let the content decide how to set the scroller. I think the modal shouldn't be responsible for that.

Reply:
yes.. that's something I thought about. Because this modal has been thought to show whatever. But for me is better! 👍  I'll update it